python/cudf_polars/cudf_polars/callback.py

```diff
@@ -135,6 +135,7 @@ def _callback(
     *,
     device: int | None,
     memory_resource: int | None,
+    executor: str | None,
```

Comment:
```suggestion
    executor: Literal["pylibcudf", "dask-experimental"] | None,
```

Reply:
Done in https://github.com/rapidsai/cudf/pull/17262/commits/2ced4a0a74a4f9c7b58ce79ea13908c686aa0f6a .